### PR TITLE
Standardize PR template across repos.

### PR DESCRIPTION
**Pull request by aekblad**
### What changes are made in this PR?
Use the same PR template as all other Spyder repos.

### Why are these changes needed?
We want our PR templates to be short and uniform.

### CODE REVIEW

**Review by cperlman — COMMENTED** [review]
this is a public repository, do we actually want this template here?

**Comment by aekblad** [comment]
> this is a public repository, do we actually want this template here?

I don't know. I do think the current PR template is overkill though.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,51 +1,9 @@
...
--->
-
-### Related issues:
-
-<!--
-Links to issues that will be closed by this PR, if any.
--->
+- [ ] This PR is linked to an issue **or** explains its purpose
+- [ ] This PR does not impact security
+- [ ] This PR does not impact functional safety
```
Comment (zbleness) [inline .github/pull_request_template.md:9]: As this is a public project maybe we can be a bit more forgiving:

```suggestion
- [ ] This PR does not impact functional safety (leave unchecked if unsure)
```

**Comment by zbleness** [comment]
> this is a public repository, do we actually want this template here?

Our internal template works here too imho. except for the functional safety check.

**Review by zbleness — APPROVED** [review]
LGTM

**Pull request merged**